src/context/auth-context.tsx

```diff
@@ -31,6 +31,19 @@ export const AuthProvider = ({ children }: { children: React.ReactNode }) => {
     }
   };
 
+  const register = async (data: RegisterData) => {
+    try {
```

Comment:
useless try/catch. You're not doing anything special with the error so you can remove the try/catch